Library/Homebrew/download_strategy.rb

```diff
@@ -492,6 +492,16 @@ def _fetch(url:, resolved_url:)
   end
 end
 
+# Download a file from a server that doesn't support download resumption,
+# also overwriting the original file's contents.
+class CurlNoResumeDownloadStrategy < CurlDownloadStrategy
```

Comment:
Would this instead take different arguments to the `CurlDownloadStrategy#initialize`? I'm a bit concerned we'll see this pop up for usage in formulae for no good reason. Better still I think this class would live in `dev-cmd/pr-pull` if it's only used there.